engine/src/flutter/.ci.yaml

```diff
@@ -457,6 +457,36 @@ targets:
       # at https://github.com/flutter/flutter/issues/152186.
       cores: "8"
 
+  - name: Linux mac_clang_tidy
```

Comment:
I think you may need to update https://github.com/flutter/flutter/blob/master/engine/src/flutter/ci/builders/mac_clang_tidy.json to include the Xcode version it won't inherit from the orchestrator anymore
https://github.com/flutter/flutter/blob/b2f515f45ef5a51362d9579fc4443cfe8f3ba387/.ci.yaml#L134-L137

Like this:

https://github.com/flutter/engine/pull/41226/files#diff-0ffa4e3b6061439de548723c6f45891936522dd624f1ea1c5567a070b39839aeR33-R37

Reply:
Thanks! Done, I ... think.

Reply:
And you might need to push the 8 core thing down but I'm not sure. Easiest is to land it and check the sub builds are getting that config.

Reply:
Sounds good, will check the node once it's merged.